packages/block-editor/src/components/global-styles/style.scss

```diff
@@ -158,6 +171,16 @@
 	cursor: pointer;
 	background: transparent;
 	border: none;
+
+
```

Comment:
Three consecutive empty new lines are not recommended, we usually [go with two](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/#structure):

> Add two blank lines between sections and one blank line between blocks in a section.